### ISSUE TITLE
avx on GCC 8 on i386 (i686), mipsel, hppa, & powerpc: mm256_insert_epi64 test failure

### ISSUE THREAD

**Issue by mr-c** [simd-everywhere/simde#643]
```
`/x86/avx/mm256_insert_epi64/native/c` [ ERROR ]
/<<PKGBUILDDIR>>/test/x86/avx.c:6941: assertion failed: r[0] == e[0] (4156894496 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)

/x86/avx/mm256_insert_epi64/native/cpp[ ERROR ]
/<<PKGBUILDDIR>>/gcc8_test/x86/avx.cpp:6941: assertion failed: r[0] == e[0] (4156894496 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)

/x86/avx/mm256_insert_epi64/emul/c   [ ERROR ]
/<<PKGBUILDDIR>>/test/x86/avx.c:6941: assertion failed: r[0] == e[0] (4156894496 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)

/x86/avx/mm256_insert_epi64/emul/cpp [ ERROR ]
/<<PKGBUILDDIR>>/gcc8_test/x86/avx.cpp:6941: assertion failed: r[0] == e[0] (4156894496 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)
```

https://buildd.debian.org/status/fetch.php?pkg=simde&arch=i386&ver=0.7.0%7Erc-1-1&stamp=1605993128&raw=0

**Comment by mr-c** [comment]
Likewise on `hppa`

```
/x86/avx/mm256_insert_epi64/native/c [ ERROR ]
/<<PKGBUILDDIR>>/test/x86/avx.c:6941: assertion failed: r[0] == e[0] (-573872016691819960 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)

/x86/avx/mm256_insert_epi64/native/cpp[ ERROR ]
/<<PKGBUILDDIR>>/gcc8_test/x86/avx.cpp:6941: assertion failed: r[0] == e[0] (-573872016691819960 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)

/x86/avx/mm256_insert_epi64/emul/c   [ ERROR ]
/<<PKGBUILDDIR>>/test/x86/avx.c:6941: assertion failed: r[0] == e[0] (-573872016691819960 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)

/x86/avx/mm256_insert_epi64/emul/cpp [ ERROR ]
/<<PKGBUILDDIR>>/gcc8_test/x86/avx.cpp:6941: assertion failed: r[0] == e[0] (-573872016691819960 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)
```

https://buildd.debian.org/status/fetch.php?pkg=simde&arch=hppa&ver=0.7.0%7Erc-1-1&stamp=1605997767&raw=0

**Comment by mr-c** [comment]
Likewise on powerpc

```
/x86/avx/mm256_insert_epi64/native/c [ ERROR ]
/<<PKGBUILDDIR>>/test/x86/avx.c:6941: assertion failed: r[0] == e[0] (-25643772252143956 == 8378794972787494426)
Error: child killed by signal 6 (Aborted)
```

https://buildd.debian.org/status/fetch.php?pkg=simde&arch=powerpc&ver=0.7.0%7Erc-1-1&stamp=1606006692&raw=0

**Comment by mr-c** [comment]
In my opinion, the 32-bit x86 part of this issue needs to be fixed or otherwise worked around before we release SIMDe 0.7.0

**Comment by nemequ** [comment]
I'm working on it, but it will probably be a couple days… I want to make sure to file bugs against the compilers, and you know how slow c-reduce is :/.

After 0.7.0 I'll try to get hppa and powerpc added to the docker development container so I can reproduce those.

**Comment by mr-c** [comment]
@nemequ 
> I'm working on it, but it will probably be a couple days… I want to make sure to file bugs against the compilers, and you know how slow c-reduce is :/.

Thanks! Even a preliminary fix would be welcome, as I can patch the pre-release Debian package and continue testing there

**Comment by mr-c** [comment]
Same error also on mipsel :-P https://buildd.debian.org/status/fetch.php?pkg=simde&arch=mipsel&ver=0.7.0%7Erc-1-3&stamp=1606674466&raw=0

**Comment by mr-c** [comment]
Since this is only a failure on gcc 8, which is rather old, I'm moving this off of the "must-fix" list for 0.7.0
